action.yaml

```diff
@@ -55,8 +63,7 @@ runs:
       - name: Assume AWS role
         uses: aws-actions/configure-aws-credentials@v1
```

Comment:
Use `v2`: https://github.com/marketplace/actions/configure-aws-credentials-for-github-actions